aloha-h2o/src/main/scala/com/eharmony/aloha/models/h2o/compiler/compiler.scala

```diff
@@ -0,0 +1,116 @@
+package com.eharmony.aloha.models.h2o.compiler
```

Comment:
Is this code you snagged from somewhere else?  If so could you provide the link.